tests/clzones_test.cpp

```diff
@@ -0,0 +1,235 @@
+#include "catch/catch.hpp"
+
+#include "clzones.h"
+#include "game_constants.h"
+#include "item.h"
+#include "item_category.h"
+#include "map.h"
+#include "map_helpers.h"
+
+static const zone_type_id zone_type_LOOT_UNSORTED( "LOOT_UNSORTED" );
+static const zone_type_id zone_type_LOOT_FOOD( "LOOT_FOOD" );
+static const zone_type_id zone_type_LOOT_PFOOD( "LOOT_PFOOD" );
+static const zone_type_id zone_type_LOOT_DRINK( "LOOT_DRINK" );
+static const zone_type_id zone_type_LOOT_PDRINK( "LOOT_PDRINK" );
+
+static void create_tile_zone( const std::string &name, const zone_type_id &zone_type, tripoint pos )
+{
+    zone_manager &zm = zone_manager::get_manager();
+    zm.add( name, zone_type, faction_id( "your_followers" ), false, true, pos, pos );
+}
+
+// Comestibles sorting is a bit awkward. Unlike other loot, they're almost
+// always inside of a container, and their sort zone changes based on their
+// shelf life and whether the container prevents rotting.
+TEST_CASE( "zone sorting comestibles ", "[zones][items][food][activities]" )
+{
+    clear_map();
+    zone_manager &zm = zone_manager::get_manager();
+
+    const tripoint origin_pos;
```

Comment:
It's easier to make it work for clang 3.8 than the other options, though (especially because that means we don't need to reexamine what compilers we support).
```suggestion
    const tripoint origin_pos = tripoint_zero;
```

Reply:
Ah, alrighty. Do I need to add something to suppress clang tidy here too or will it be fine once it's pushed?

Reply:
AFAIK clang-tidy will be fine with this, but we'll see if it gets cranky.

Reply:
For future reference, I think the way you can appease both clang-tidy and clang in this situation is to initialize it like
```
const tripoint origin_pos{};
```